test/specs/commonTests.js

```diff
@@ -468,6 +468,13 @@ export const rendersChildren = (Component, options = {}) => {
     shallow(createElement(Component, requiredProps, child))
       .should.contain(child)
   })
+
+  if (Component._meta.name !== 'Embed') {
```

Comment:
Let's remove this check. You've have failure because [`Embed`](https://github.com/Semantic-Org/Semantic-UI-React/blob/2bfaa3563d3dde230ad54b0e1bdaedba04bf43af/test/specs/modules/Embed/Embed-test.js#L23) asserts use `requiredProps` option:

```js
  common.rendersChildren(Embed, {
    requiredProps: { active: true },
  })
```

Lets use factory there:

```diff
- it('renders child number with 0 value', ()=> {
-   const wrapper = shallow(<Component>{0}</Component>)
-   wrapper.text().should.equal('0')
- })
+ it('renders child number with 0 value', () => {
+ shallow(createElement(Component, requiredProps, 0))
+   .should.contain.text('0')
+ })
```